src/__tests__/__snapshots__/toNestObject.ts.snap

```diff
@@ -10,15 +10,25 @@ Object {
     },
     "type": "st",
   },
-  "test": Array [
-    Object {
+  "test": Object {
+    "0": Object {
```

Comment:
Why has this becoming object?

Reply:
I believe it's because the snapshot serializer serializes arrays with non-numeric properties as objects.

Reply:
That's because the [test error is an object](https://github.com/react-hook-form/resolvers/pull/442/files#diff-e8bad6d3d67175a94e94db1c8258febe5a4017bfa4908c1948a58e72570b07cfR6).